_episodes/documentation.md

```diff
@@ -1,33 +1,92 @@
 ---
 title: "Documentation"
-teaching: 0
-exercises: 0
+teaching: 5
+exercises: 15
 questions:
-- "Key question (FIXME)"
+- "What is documentation?"
+- "Where to document my data?"
+- "What is the difference between documentation and metadata?"
 objectives:
-- "First learning objective. (FIXME)"
+- "Evaluate existing data documentation with a critical view towards reusability"
```

Comment:
Not sure this objective is covered in the lesson.

Reply:
Well, it's one of the exercises. Perhaps we can add a checklist somewhere. I found a nice one [here](https://zenodo.org/record/3405141#.X32Tn-1R1hE) but maybe that one is better at the end of the lesson? I'll make an issue to add it somewhere (later).

Reply:
#20